### PR TITLE
Fixes the URL generation for the geocoding & import models

### PR DESCRIPTION
**Pull request by javierarce**
The URLs used to check the status of the imports and geocodings weren't adding the `baseURL` component in the path, and therefore failing in some cases.

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @viddo

File: lib/assets/javascripts/cartodb3/data/background-importer/geocoding-model.js [hunk 1]
```diff
@@ -35,13 +35,13 @@ module.exports = cdb.core.Model.extend({
...
-    var base = '/api/' + version + '/geocodings/';
+    var baseUrl = this._configModel.get('base_url');
+    var geocodingsPath = '/api/' + version + '/geocodings';
 
     if (this.isNew()) {
-      return base;
+      return baseUrl + geocodingsPath;
     }
-    return base + this.id;
+    return baseUrl + geocodingsPath + '/' + this.id;
```
Comment (viddo) [inline lib/assets/javascripts/cartodb3/data/background-importer/geocoding-model.js]: I know this model was ported from the old code, but I think it's a good opportunity to fix this kind of maluse of url methods; define the method as [urlRoot](http://backbonejs.org/#Model-urlRoot) instead and you don't have to check isNew and append the id manually

File: lib/assets/javascripts/cartodb3/data/background-importer/import-model.js [hunk 2]
```diff
@@ -23,12 +23,14 @@ module.exports = cdb.core.Model.extend({
...
+    var baseUrl = this._configModel.get('base_url');
+
+    var importsPath = '/api/' + version + '/imports';
 
     if (this.isNew()) {
-      return base;
+      return baseUrl + importsPath;
     }
-    return base + '/' + this.id;
+    return baseUrl + importsPath + '/' + this.id;
```
Comment (viddo) [inline lib/assets/javascripts/cartodb3/data/background-importer/import-model.js]: (Ditto above) I know this model was ported from the old code, but I think it's a good opportunity to fix this kind of maluse of url methods; define the method as [urlRoot](http://backbonejs.org/#Model-urlRoot) instead and you don't have to check isNew and append the id manually

**Comment by viddo** [comment]
👍

File: lib/assets/javascripts/cartodb3/data/background-importer/geocoding-model.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/data/background-importer/geocoding-model.js]: Oh, true. Thanks!

File: lib/assets/javascripts/cartodb3/data/background-importer/import-model.js (hunk 2 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb3/data/background-importer/import-model.js]: 👌

**Pull request merged**